src/vars/styles/styles.js

```diff
@@ -0,0 +1,8 @@
+import { css } from '@emotion/react';
+
+export const dashboardPanels = css`
```

Comment:
So `emotion` actually supports inheritance, which is what I was doing with these in `WidgetContainer` (in `components/Dashboard/widgets/common.tsx`).

We can go with whichever style, I'm not sure I have a preference, but either way we should probably stick to one when reasonable (as it is in this case).

Reply:
oh, cool ! I am on it. Will change to inheritance, I didnt notice that part :)